decentraland/components/presentational/LandForSaleInfo.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { StyleSheet, View, Text, Alert } from "react-native";
+import PropTypes from "prop-types";
+import {
+  responsiveWidth,
+  responsiveFontSize,
+} from "react-native-responsive-dimensions";
+import Colors from "@constants/Colors";
+import { Button, Icon } from "native-base";
+import { formatNumber } from "../../helpers";
+
+const onPriceInfo = () => {
+  const title = "";
+  const message =
+    "MANA is Decentraland’s fungible, ERC20 cryptocurrency token. MANA is burned, or spent, in exchange for LAND parcels. ";
```

Comment:
```suggestion
    "MANA is Decentraland’s fungible, ERC20 cryptocurrency token. MANA is burned or spent in exchange for LAND parcels. ";
```

Reply:
I wonder if "fungible" is too confusing of a word to show to the more mainstream audience we're targeting. I'm thinking it is too confusing.

Reply:
Extracted from here: https://docs.decentraland.org/decentraland/faq/#what-is-mana

Reply:
Good to have the source info - thanks! The comma usage needed to be corrected, though.

Reply:
Sure!

Reply:
How do you feel about just `MANA is Decentraland’s cryptocurrency token.` for the first sentence?

Reply:
I like the last change: `MANA is Decentraland’s cryptocurrency token. You can use MANA to buy LAND parcels.`

Reply:
Great! Resolving